src/main/kotlin/org/malachite/estella/commons/EStellaHeaders.kt

```diff
@@ -5,4 +5,5 @@ object EStellaHeaders{
     const val refreshToken = "X-Refresh-Token"
     const val authToken = "X-Auth-Token"
     const val adminApiKey = "X-Admin-Api-Key"
+    const val hrPartnerMail = "X-hr-mail"
```

Comment:
Widzę, ze tego headera używasz tylko w jednym miejscu. Nie lepiej przekazać tego maila przez body? Wydaje mi się, że każda metoda http może mieć body, więc dlaczego by nie skorzystać.

Reply:
GET nie może z tego co pamiętam

Reply:
W elasticu może, także ogólnie chyba też XDDD

Reply:
Czytałeś update odpowiedzi ? Że obecnie niektóre implementację odrzucają get request jeśli ma body?

Reply:
Nie czytałem, ale testowałem na localhoscie po zmianie na body i działało

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET
Znaczy to jest możliwe do wykonania ale tego się nie powinno robić XD To tak samo jak można tworzyć osobne nazwy endpointow na posta typu addUser etc

Reply:
Ale ale jak chcecie to tak zaklepać no to ok

Reply:
Nie no ja tez nie jestem przekonany to wersji z body, wiec ja mogę bez problemu to cofnąć xD